turbopack/crates/turbo-tasks/src/scope.rs

```diff
@@ -0,0 +1,50 @@
+use std::sync::Arc;
+
+use crate::{turbo_tasks, turbo_tasks_scope, TurboTasksApi};
+
+pub struct Scope<'scope, 'a> {
+    scope: &'a rayon::Scope<'scope>,
+    handle: tokio::runtime::Handle,
+    turbo_tasks: Arc<dyn TurboTasksApi>,
+    span: tracing::Span,
+}
+
+impl<'scope, 'a> Scope<'scope, 'a> {
+    pub fn spawn<BODY>(&self, body: BODY)
+    where
+        BODY: FnOnce(&Scope<'scope, '_>) + Send + 'scope,
+    {
```

Comment:
Naming conventions for type parameters is `UpperCamelCase`: https://rust-lang.github.io/api-guidelines/naming.html
```suggestion
    pub fn spawn<Body>(&self, body: Body)
    where
        Body: FnOnce(&Scope<'scope, '_>) + Send + 'scope,
    {
```